### DISCUSSION TITLE
Is it possible to make the topmost block invisible?

### DISCUSSION THREAD

**Discussion by EasyDevv** [leodevbro/vscode-blockman#129] [Q&A] [answered]
![image](https://github.com/leodevbro/vscode-blockman/assets/11148818/8cf85e54-fbe5-4efb-9c75-2fbba561d26f)

Can I make the first block invisible?

**Comment by leodevbro** [comment, marked as answer]
Yes, it's possible and it's easy. Please see this comment:

https://github.com/leodevbro/vscode-blockman/issues/101#issuecomment-1439499877

**Reply by EasyDevv** [reply]
Thanks, it works fine
